File: samples/java/DeliveryDispatch/Server/Configuration/src/main/java/systems/zlink/samples/deliverydispatch/server/configuration/SampleNames.java
package systems.zlink.samples.deliverydispatch.server.configuration;

public final class SampleNames {
    private SampleNames() {}

    public static final String CourierChannel = "deliverydispatch.courier";

    /**
     * Where the courier's decision comes back to. The offer goes out one-way and the decision
     * returns as its own one-way message, so dispatch has to be reachable as a channel server
     * (common sample spec section 7.4).
     */
    public static final String DispatchChannel = "deliverydispatch.dispatch";

    public static final String TrackingChannel = "deliverydispatch.tracking";
    public static final String CustomerSpotDiscovery = "delivery-customers";
    public static final String CourierSpotDiscovery = "delivery-couriers";
    public static final String CustomerStreamNode = "deliverydispatch.customer.stream";
    public static final String CourierStreamNode = "deliverydispatch.courier.stream";
    public static final String CustomerActorType = "deliverydispatch.customer.actor";
    public static final String CourierActorType = "deliverydispatch.courier.actor";

    public static final String DispatchNode = "dispatch";
    public static final String TrackingNode = "tracking";
    public static final String CourierNode1 = "courier-node-1";
    public static final String CourierNode2 = "courier-node-2";
    public static final String CustomerGatewayNode = "customer-gateway";
    public static final String CourierSessionNode = "courier-session";

    public static final String ReassignmentMarker = "deliverydispatch-reassignment=completed";
    public static final String ServerEvidenceMarker = "deliverydispatch-server-evidence=completed";
    public static final String CompletedMarker = "deliverydispatch=completed";
}

File: samples/kotlin/DeliveryDispatch/Server/Configuration/src/main/kotlin/systems/zlink/samples/kotlin/deliverydispatch/server/configuration/SampleNames.kt
package systems.zlink.samples.kotlin.deliverydispatch.server.configuration

object SampleNames {
    const val CourierChannel: String = "deliverydispatch.courier"

    /**
     * Where the courier's decision comes back to. The offer goes out one-way and the decision
     * returns as its own one-way message, so dispatch has to be reachable as a channel server
     * (common sample spec section 7.4).
     */
    const val DispatchChannel: String = "deliverydispatch.dispatch"
    const val TrackingChannel: String = "deliverydispatch.tracking"
    const val CustomerSpotMesh: String = "delivery-customers"
    const val CourierSpotMesh: String = "delivery-couriers"
    const val CustomerStreamNode: String = "deliverydispatch.customer.stream"
    const val CourierStreamNode: String = "deliverydispatch.courier.stream"
    const val CustomerActorType: String = "deliverydispatch.customer.actor"
    const val CourierActorType: String = "deliverydispatch.courier.actor"

    const val RegistryRole: String = "registry"
    const val DispatchRole: String = "dispatch"
    const val CourierSessionRole: String = "couriersession"
    const val CourierSpotNodeRolePrefix: String = "courierspotnode"
    const val TrackingRole: String = "tracking"
    const val CustomerGatewayRole: String = "customergateway"

    const val ReassignmentMarker: String = "deliverydispatch-reassignment=completed"
    const val ServerEvidenceMarker: String = "deliverydispatch-server-evidence=completed"
    const val CompletedMarker: String = "deliverydispatch=completed"

    const val DispatchNode: String = "dispatch"
    const val TrackingNode: String = "tracking"
    const val CourierNode1: String = "courier-node-1"
    const val CourierNode2: String = "courier-node-2"
    const val CustomerGatewayNode: String = "customer-gateway"
    const val CourierSessionNode: String = "courier-session"
}
